packages/x-charts-pro/src/ChartZoomSlider/internals/ChartAxisZoomSlider.tsx

```diff
@@ -141,6 +147,140 @@ export function ChartAxisZoomSlider({ axisDirection, axisId }: ChartZoomSliderPr
   );
 }
 
+interface ChartAxisZoomSliderTrackProps extends React.ComponentProps<'rect'> {
+  axisId: AxisId;
+  axisDirection: 'x' | 'y';
+  reverse: boolean;
+}
+
+function ChartAxisZoomSliderTrack({
+  axisId,
+  axisDirection,
+  reverse,
+  ...other
+}: ChartAxisZoomSliderTrackProps) {
+  const ref = React.useRef<SVGRectElement>(null);
+  const { instance, svgRef } = useChartContext<[UseChartProZoomSignature]>();
+  const store = useStore<[UseChartProZoomSignature]>();
+
+  const onPointerDown = function onPointerDown(event: React.PointerEvent<SVGRectElement>) {
+    const rect = ref.current;
+    const element = svgRef.current;
+
+    if (!rect || !element) {
+      return;
+    }
+
+    const pointerDownPoint = getSVGPoint(element, event);
+    let zoomFromPointerDown = calculateZoomFromPoint(store.getSnapshot(), axisId, pointerDownPoint);
```

Comment:
maybe something like `firstZoomPoint` and `currentZoomPoint` for the move one?

Reply:
or `initialZoomPoint`

Reply:
I'm not sure if `firstZoomPoint` is better because this isn't a point, it's the zoom value (ranging from 0 to 100) of the pointer down event. I could call it `initialZoom`, but not sure if that's less confusing 🤔

Reply:
I'll merge this PR as it's been approved, but we can keep discussing and I'll update the code in a follow-up PR.

Reply:
My line of thought was that it is a point in the zoom "scale"

Reply:
Yeah, we can look at it that way. I'm afraid it might be confused with the `SVGPoint` that we're using in the same scope, that's why I initially tried suffixing it with `Zoom`